scripts/release

```diff
@@ -586,7 +586,9 @@ generate_release_notes() {
     path="$2"
     git diff-tree --no-commit-id --name-only -r "$sha" \
       | grep -v "$path" \
-      | grep -v "MaterialComponents.podspec"
+      | grep -v "MaterialComponents.podspec" \
```

Comment:
Do you want to exclude BUILD files?

Reply:
BUILD files are within the component's directory so we generally don't have to ignore them. This doesn't include the root BUILD file, but as mentioned in the PR description I think there may want to pursue a more generalizable approach here.